edi_notification_oca/tests/test_edi_notification.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2024 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+
+import base64
+
+from odoo.addons.edi_oca.tests.common import EDIBackendCommonComponentRegistryTestCase
+from odoo.addons.edi_oca.tests.fake_components import FakeInputProcess
+
+
+class TestEDINotification(EDIBackendCommonComponentRegistryTestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls._setup_env()
+        cls._build_components(
+            cls,
+            FakeInputProcess,
+        )
+        cls._load_module_components(cls, "edi_notification_oca")
+        vals = {
+            "model": cls.partner._name,
+            "res_id": cls.partner.id,
+            "exchange_file": base64.b64encode(b"1234"),
+        }
+        cls.record = cls.backend.create_record("test_csv_input", vals)
+        cls.group_sale = cls.env.ref("sales_team.group_sale_salesman")
+        cls.user_a = cls._create_user(cls, "A")
+        cls.user_b = cls._create_user(cls, "B")
+        cls.user_c = cls._create_user(cls, "C")
+
+    def setUp(self):
+        super().setUp()
+        FakeInputProcess.reset_faked()
+
+    def _create_user(self, letter):
+        return self.env["res.users"].create(
```

Comment:
```suggestion
        return self.env["res.users"].with_context(no_reset_password=True).create(
```
Always :wink: 
Or... use `new_test_user` from `odoo.tests.common`.